[2024-02-12 14:06:47,909] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: sales_mart_inc.update_d_city scheduled__2024-02-07T14:01:59.240675+00:00 [queued]>
[2024-02-12 14:06:47,913] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: sales_mart_inc.update_d_city scheduled__2024-02-07T14:01:59.240675+00:00 [queued]>
[2024-02-12 14:06:47,913] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-02-12 14:06:47,914] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2024-02-12 14:06:47,914] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-02-12 14:06:47,919] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): update_d_city> on 2024-02-07 14:01:59.240675+00:00
[2024-02-12 14:06:47,921] {standard_task_runner.py:52} INFO - Started process 28875 to run task
[2024-02-12 14:06:47,923] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'sales_mart_inc', 'update_d_city', 'scheduled__2024-02-07T14:01:59.240675+00:00', '--job-id', '564', '--raw', '--subdir', 'DAGS_FOLDER/sales_mart_inc.py', '--cfg-path', '/tmp/tmp_9qmyzq6', '--error-file', '/tmp/tmprn3fjjxr']
[2024-02-12 14:06:47,923] {standard_task_runner.py:80} INFO - Job 564: Subtask update_d_city
[2024-02-12 14:06:47,953] {task_command.py:369} INFO - Running <TaskInstance: sales_mart_inc.update_d_city scheduled__2024-02-07T14:01:59.240675+00:00 [running]> on host 91766ce79ef0
[2024-02-12 14:06:47,987] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=student@example.com
AIRFLOW_CTX_DAG_OWNER=student
AIRFLOW_CTX_DAG_ID=sales_mart_inc
AIRFLOW_CTX_TASK_ID=update_d_city
AIRFLOW_CTX_EXECUTION_DATE=2024-02-07T14:01:59.240675+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-02-07T14:01:59.240675+00:00
[2024-02-12 14:06:47,993] {base.py:68} INFO - Using connection ID 'postgresql_de' for task execution.
[2024-02-12 14:06:47,996] {sql.py:315} INFO - Running statement: insert into mart.d_city (city_id, city_name)
select city_id, city_name from staging.user_order_log
where city_id not in (select city_id from mart.d_city)
group by city_id, city_name;, parameters: None
[2024-02-12 14:06:47,997] {sql.py:324} INFO - Rows affected: 0
[2024-02-12 14:06:48,005] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=sales_mart_inc, task_id=update_d_city, execution_date=20240207T140159, start_date=20240212T140647, end_date=20240212T140648
[2024-02-12 14:06:48,015] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-02-12 14:06:48,036] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
